app/views/articles/new.html.erb

```diff
@@ -2,4 +2,4 @@
 
 <br>
 
-<%= render 'form', model: @article %>
\ No newline at end of file
+<%= render 'articles/partials/form', model: @article %>
```

Comment:
I think you can just put "render partial: "form", model: @article" or does the fail saying that it can't find the file because its in the partials folder?